llvm/utils/TableGen/DAGISelEmitter.cpp

```diff
@@ -165,8 +165,21 @@ void DAGISelEmitter::run(raw_ostream &OS) {
   // Add all the patterns to a temporary list so we can sort them.
   Records.startTimer("Sort patterns");
   std::vector<const PatternToMatch *> Patterns;
-  for (const PatternToMatch &PTM : CGP.ptms())
+  for (const PatternToMatch &PTM : CGP.ptms()) {
+
+    // Disable import of patterns marked as ignore.
+    const TreePatternNode &Dst = PTM.getDstPattern();
+    if (!Dst.isLeaf()) {
+      const Record *Op = Dst.getOperator();
+      const bool shouldIgnore =
```

Comment:
`CamelCase`

Reply:
Thanks. [Here](https://github.com/llvm/llvm-project/pull/88382/commits/4a727fb1b7e67c4d4601afc98f29fa4c57bc4ef9#diff-6ad0fdb2565eaadd898ed286afe1367486fc0ea42880ba5cb17e8e594fccdb2d)'s the edit.